foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/databaseaccess/DatabasePlatform.java

```diff
@@ -2684,7 +2685,7 @@ public int appendParameterInternal(Call call, Writer writer, Object parameter) {
                 java.sql.Timestamp ts = java.sql.Timestamp.valueOf(java.time.LocalDateTime.of(java.time.LocalDate.ofEpochDay(0), ot.toLocalTime()));
                 appendTimestamp(ts, writer);
             } else if (dbValue instanceof java.time.Year){
-                appendNumber((Number)dbValue, writer);
+                appendNumber(((java.time.Year) dbValue).getValue(), writer);
```

Comment:
With the Year imported - perhaps FQCN is not needed here?

Reply:
I want keep there same pattern and rest of code in the `DatabasePlatform` class and to be sure (avoid any conflict in the future), that "right" `Year` class is used.